az_quickstart.install

```diff
@@ -217,3 +217,10 @@ function az_quickstart_update_1020901(&$sandbox) {
   $sandbox['#finished'] = empty($sandbox['max']) ? 1 : $sandbox['progress'] / $sandbox['max'];
   return t('Updated langcode on %count menu links.', ['%count' => $sandbox['progress']]);
 }
+
+/**
+ * Enable masquerade module by default.
+ */
+function az_quickstart_update_920703() {
```

Comment:
```suggestion
function az_quickstart_update_1021301() {
```
We'll need to change the number used in this update function name to match the Quickstart version we want to introduce this change for.  (See https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#update-hook-numbering-convention)